src/test/java/api/support/matchers/JsonPathMatchers.java

```diff
@@ -0,0 +1,43 @@
+package api.support.matchers;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import com.jayway.jsonpath.JsonPath;
+
+import io.vertx.core.json.JsonObject;
+
+public class JsonPathMatchers {
+
+  private JsonPathMatchers() {}
+
+  public static <T> Matcher<JsonObject> hasJsonPath(String path, Matcher<T> valueMatcher) {
```

Comment:
What are benefits to use "new" hasJsonPath method instead of com.jayway.jsonpath.matchers.JsonPathMatchers#hasJsonPath method which we use in other test methods?

Reply:
Unfortunately the jayway matcher does not support `JsonObject` object. But let me simplify the matcher a bit, so it will delegate matching to the jaway one.